src/Main.java

```diff
@@ -37,21 +38,23 @@ public static String findEmploeerWithMinSalary() {
         int emp = 0;
         for (int i = 1; i < employees.length; i++) {
             if (minimum > employees[i].getSalary()) {
+                minimum = employees[i].getSalary();
                 emp = i;
             }
         }
-        return employees[emp].getFio();
+        return employees[emp].toString();
     }
 
     public static String findEmploeerWithMaxSalary() {
         int maximum = employees[0].getSalary();
         int emp = 0;
         for (int i = 1; i < employees.length; i++) {
             if (maximum < employees[i].getSalary()) {
+                maximum = employees[i].getSalary();
                 emp = i;
             }
         }
-        return employees[emp].getFio();
+        return employees[emp].toString();
```

Comment:
```suggestion
        return employees[emp];
```
надо вернуть сотрудника, а не строку